mkdocs.yml

```diff
@@ -55,6 +55,9 @@ nav:
               - Data Format: develop/Data_Format.md
               - Election Record: develop/Election_Record.md
               - Sample Data: develop/Sample_Data.md
+      - Events:
+          - Monthly Office Hours: events/Office_hours.md
```

Comment:
Maybe we should start using lower case for file names. I've been debating on this. Safer representation of the actual paths.

Reply:
works for me!